service/src/worker.rs

```diff
@@ -169,18 +161,7 @@ where
 			))
 			.unwrap();
 			trace!("found peer rpc url: {}", enclave_url);
-			let worker_api_direct = DirectWorkerApi::new(enclave_url.clone().into());
-			match worker_api_direct.get_untrusted_worker_url() {
-				Ok(untrusted_worker_url) => {
-					peer_urls.push(untrusted_worker_url);
-				},
-				Err(e) => {
-					error!(
-						"Failed to get untrusted worker url (enclave: {}): {:?}",
-						enclave_url, e
-					);
-				},
-			}
+			peer_urls.push(enclave_url.into());
```

Comment:
Just commenting for the records, we change the to `enclave_url` because the server is now running on the trusted side.